data/property-sort-orders/smacss.txt

```diff
@@ -121,41 +167,66 @@ color
 font
 font-family
 font-size
+font-size-adjust
+font-stretch
+font-effect
+font-smooth
 font-smoothing
+-moz-osx-font-smoothing
+-webkit-font-smoothing
```

Comment:
Good call. I suggest you look into using something like [autoprefixer](https://github.com/postcss/autoprefixer) to avoid having to use vendor-prefixes.